### ISSUE TITLE
Need more info, please

### ISSUE THREAD

**Issue by NewGrafon** [EXBO-Studio/stalcraft-database#1]
Need info about weapon classes upgrades and info about artefacts additional properties
Weapon classes upgrades can be presented like this:
weaponClass: {
    lvl1: {
        damage: '+0.73',
        spread: '-0.33'
    },
    lvl2: {
        damage: '+1.62',
        spread: '-0.72'
    },
    ...lvlN: {
        damage: '+x.xx',
        spread: '-k.kk'
    }...
    lvl15: {
        damage: '+30',
        spread: '-15'
    }
}

**Comment by GBLightning** [comment]
Yep, it would be absolute awesome to have invormation about artifacts additional properties in the DB.
